shoop/front/apps/customer_information/views.py

```diff
@@ -40,7 +42,7 @@ class CustomerEditView(FormView):
     template_name = "shoop/customer_information/edit.jinja"
 
     def get_form(self, form_class):
-        contact = get_person_contact(self.request.user)
+        contact = get_company_contact(self.request.user) or get_person_contact(self.request.user)
```

Comment:
In my opinion this is not part of this task since you should only enable login for person contacts that is linked to companies.